changelogs/changelog.yaml

```diff
@@ -706,3 +706,44 @@ releases:
     fragments:
     - bump_release_version.yml
     release_date: '2022-02-15'
+  3.2.0:
+    changes:
+      breaking_changes:
+      - aws_ssm - on_denied and on_missing now both default to error, for consistency
+        with both aws_secret and the base Lookup class (https://github.com/ansible-collections/amazon.aws/issues/617).
```

Comment:
Should we have backported a *breaking* change?

Reply:
I thought we could include breaking changes in 3.2.0. What do you suggest to do? Do we want to revert it and include in 4.0.0 directly?

Reply:
Uh yes. I think it must be reverted vom `stable-3`

https://github.com/ansible-collections/amazon.aws/pull/618#issuecomment-1034254967

Reply:
Can I do it in this PR or do we need a separate one?

Reply:
Sorry, missed completely that comment.

Reply:
If possible I would give it a try.

```
git reset --hard aeba19a # commit before that merge
git cherry-pick 7297e31 c06d77c 42295a7 d9d5d0d # cherry pick what comes after
```

and push with force? But I've no idea if that's more a punk way and will result in merge conflicts?

Reply:
Ok, thanks. let me try.

Reply:
...plus rebuild docs and changelog.  
naybe there is also a smoother "rebase" way.....

Reply:
I rebuilt docs, but I have to hard reset and do it from the beginning. It happen in the past the same thing with the docs. I am re-doing right now. I only used `git revert commit_hash`

Reply:
I then need to update the version_added to 4.0.0 for #618